app/bundles/CoreBundle/Templating/Twig/Extension/AssetExtension.php

```diff
@@ -113,4 +114,9 @@ public function addScriptDeclaration(string $script, string $location = 'head'):
     {
         return $this->assetsHelper->addScriptDeclaration($script, $location);
     }
+
+    public function getCountryFlag($country, $urlOnly = true, $class = '')
```

Comment:
We have PHPSTAN configured to require input and output types for new methods. Please add them like this:
```suggestion
    public function getCountryFlag(string $country, bool $urlOnly = true, string $class = ''): string
```